openwisp-utils-qa-checks

```diff
@@ -42,36 +43,43 @@ runcheckendline() {
     fi
   done
   if [ $flag -ne 0 ]; then
-    exit 1
+    echoerr "ERROR: Blank endline check failed!"
+    FAILURE=1
+  else
+    echo "SUCCESS: Blank endline check successful!"
   fi
 }
 
 runcheckmigrations() {
-  echo "Running migration name check"
   if [ "$MIGRATION_PATH" == "" ]; then
-    echo "No migration path specified!"
-    exit 5
+    echoerr "No migration path specified!"
+    FAILURE=1
+    return
   fi
-  checkmigrations --migration-path "$MIGRATION_PATH" --migrations-to-ignore "$MIGRATIONS_TO_IGNORE" || exit 2
+  checkmigrations --migration-path "$MIGRATION_PATH" --migrations-to-ignore "$MIGRATIONS_TO_IGNORE" \
+    && echo "SUCCESS: Migration name check successful!" \
+    || { echoerr "FAIL: Migration name check failed!"; FAILURE=1; }
```

Comment:
why did you use `FAIL:` here while in the others error messages you used `ERROR:`? 
Let's keep consistency, I like `ERROR: ` because it's very explicit.